test/functional/feature_assumeutxo.py

```diff
@@ -152,6 +148,35 @@ def test_invalid_mempool_state(self, dump_output_path):
 
         self.restart_node(2, extra_args=self.extra_args[2])
 
+    def test_snapshot_in_a_divergent_chain(self, dump_output_path):
+        # First rollback node2's chain to the pregenerated one, up to height 199
+        node = self.nodes[2]
+        block_hash = node.getblockhash(START_HEIGHT + 1)
+        node.invalidateblock(block_hash)
+        assert_equal(node.getblockcount(), START_HEIGHT)
+
+        self.log.info(f"Check importing a snapshot where current chain-tip is not an ancestor of the snapshot block but has less work")
+        # Generate a divergent chain in node2 but with less work compared to the snapshot
+        self.generate(node, nblocks=99, sync_fun=self.no_op)
```

Comment:
`nblocks=99`
We can make this value dependent on SNAPSHOT_BASE_HEIGHT such as (SNAPSHOT_BASE_HEIGHT - 200) or (SNAPSHOT_BASE_HEIGHT / 2) so that it's tied to SNAPSHOT_BASE_HEIGHT, and any future changes will automatically account for this variable as well.

Reply:
Good point. The goal of this test is to generate a divergent chain starting from `START_HEIGHT` but having less work than the snapshot, which height equals `SNAPSHOT_BASE_HEIGHT`. Maybe I can set `nBlocks=SNAPSHOT_BASE_HEIGHT-START_HEIGHT-1`. This will result in `99` and it will adapt to any future changes to either `START_HEIGHT` or `SNAPSHOT_BASE_HEIGHT`

Reply:
Yes, that would be good.

Reply:
Done. Thanks